cabalscan/src/CabalScan/RuleGenerator.hs

```diff
@@ -384,10 +379,10 @@ data UnresolvedCabalDependencies = UnresolvedCabalDependencies
   }
   deriving (Show, Exception)
 
-readCabalFile :: Path b File -> IO Cabal.PackageDescription
+readCabalFile :: FilePath -> IO Cabal.PackageDescription
 readCabalFile cabalFilePath = do
-  let cabalFile = Path.toFilePath cabalFilePath
-  genericPkg <- Cabal.readGenericPackageDescription Verbosity.normal cabalFile
+  let cabalFile = cabalFilePath
+  genericPkg <- Cabal.readGenericPackageDescription Cabal.silent cabalFile
```

Comment:
Because the meaning of normal verbosity apparently changed between the version of cabal included in the snapshots lts-18.. and lts-19...
See for instance https://github.com/tweag/gazelle_cabal/actions/runs/2575589850 where the standard output is polluted with Warning... making it not a valid Json file anymore.